firebase-vscode/package.json

```diff
@@ -14,7 +14,7 @@
     "Other"
   ],
   "extensionDependencies": [
-    "graphql.vscode-graphql-syntax"
+    "GraphQL.vscode-graphql-syntax"
```

Comment:
<img width="218" alt="Screenshot 2024-09-27 at 12 09 41 PM" src="https://github.com/user-attachments/assets/f0718822-033f-4d39-be13-041531b31e68">
It kinda looks like this is supposed to be lowercase, no?

Reply:
If you copy the id it's capitalized in both VSCode and open-vsx.  This fix is targeted at fixing open-VSX builds. Locally it works fine and installs the dependency.